packages/contractkit/.gitignore

```diff
@@ -3,4 +3,5 @@ lib/
 .devchain/
 .devchain.tar.gz
 tmp/
-.tmp/
\ No newline at end of file
+.tmp/
+.env
```

Comment:
I included a template for devs (above). The expectation is that each dev will modify their local copy with the credentials. Since we shouldn't be checking this in (contains secrets), I added it to the .gitignore to prevent tracking of future changes.